meetupselector/api/routes/users.py

```diff
@@ -1,12 +1,16 @@
 from http import HTTPStatus
 
+from django.conf import settings
+from django.urls import reverse
 from ninja import Router
 from ninja.security import django_auth
 
 from meetupselector.api.schemas.users import LoginSchema
 from meetupselector.user.schemas import SignInSchema
 from meetupselector.user.services import UserService
 
+confirmation_url_name = "signin_confirmation"
```

Comment:
As you import `settings`, why don't put this variable in settings?

Reply:
moved to settings